src/test/java/reposense/report/RepoClonerTest.java

```diff
@@ -22,4 +31,17 @@ public void repoCloner_emptyRepo_failsGracefully() throws Exception {
 
         Assert.assertNull(clonedRepoLocation);
     }
+
+    @Test
+    public void repoCloner_validRepoLocationWithRelativePathingAndSpaces_success() throws Exception {
+        //Clones a test repository into the test directory for testing of relative pathing
```

Comment:
```suggestion
        // Clones a test repository into the test directory for testing of relative pathing
```

Adds an extra space after the comment marker.

Reply:
Thanks for catching it. Change has been committed.